index.js

```diff
@@ -12,23 +12,31 @@ start();
 async function start() {
   let min = 1;
   let max = 100;
+  let guessArray = []
```

Comment:
guessArray is never really used in the game. Consider removing to simplify the code